py3status/modules/external_script.py

```diff
@@ -21,7 +22,12 @@
         (default False)
 
 Format placeholders:
-    {output} output of script given by "script_path"
+    {line} number of lines in the output
+    {output} first line of the output of script given by "script_path"
+
+Format notification placeholders:
+    {line} number of lines in the output
+    {output} first line of the output of script given by "script_path"
```

Comment:
Fixed documentation here, `{output_full}` still remains undocumented as per your request.

Reply:
Why don't document this?

Reply:
Documented. :sob: